proxy/ControlBase.cc

```diff
@@ -585,12 +586,12 @@ SuffixMod::check(HttpRequestData *req) const
   int path_len;
   const char *path = req->hdr->url_get()->path_get(&path_len);
 
-  if (1 == static_cast<int>(this->text_vec.count()) && 1 == static_cast<int>(this->text_vec[0].size()) &&
+  if (1 == static_cast<int>(this->text_vec.size()) && 1 == static_cast<int>(this->text_vec[0].size()) &&
```

Comment:
Are the static_cast redundant here?

Reply:
@dragon512 well this wasn't out of scope to be changed so I modified only access to the class instance methods